mod_reforged/hooks/skills/racial/ghost_racial.nut

```diff
@@ -0,0 +1,24 @@
+::mods_hookExactClass("skills/racial/ghost_racial", function(o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+		this.m.Name = "Ghost";
+		this.m.Icon = "ui/orientation/ghost_01_orientation.png";
+		this.m.IsHidden = false;
+	}
+
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
```

Comment:
Minor comment: `.push` should be preferred here as you are only pushing a single entry.